notebooks/data analysis/Pathfinder_system_abilities_attributes.ipynb

```diff
@@ -0,0 +1,1915 @@
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "<class 'pandas.core.series.Series'>\n",
+      "Int64Index: 1206 entries, 0 to 364\n",
+      "Series name: hardness\n",
+      "Non-Null Count  Dtype \n",
+      "--------------  ----- \n",
+      "10 non-null     object\n",
+      "dtypes: object(1)\n",
+      "memory usage: 51.1+ KB\n"
+     ]
+    }
+   ],
+   "source": [
+    "attributes.hardness.info()"
+   ],
+   "metadata": {
+    "collapsed": false
+   }
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 245,
+   "outputs": [
+    {
+     "data": {
+      "text/plain": "0    NaN\n1    NaN\n2    NaN\n3    NaN\n4    NaN\nName: hardness, dtype: object"
+     },
+     "execution_count": 245,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "attributes.hardness.head()"
+   ],
+   "metadata": {
+    "collapsed": false
+   }
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 247,
+   "outputs": [],
+   "source": [
+    "hardness = unpack_column(attributes[pd.notnull(attributes.hardness)], \"hardness\")"
+   ],
+   "metadata": {
+    "collapsed": false
+   }
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 248,
+   "outputs": [
+    {
+     "data": {
+      "text/plain": "     value\n109      2\n185      6\n286      9\n436     10\n306     10\n17      15\n56       8\n251     10\n253     14\n351      3",
+      "text/html": "<div>\n<style scoped>\n    .dataframe tbody tr th:only-of-type {\n        vertical-align: middle;\n    }\n\n    .dataframe tbody tr th {\n        vertical-align: top;\n    }\n\n    .dataframe thead th {\n        text-align: right;\n    }\n</style>\n<table border=\"1\" class=\"dataframe\">\n  <thead>\n    <tr style=\"text-align: right;\">\n      <th></th>\n      <th>value</th>\n    </tr>\n  </thead>\n  <tbody>\n    <tr>\n      <th>109</th>\n      <td>2</td>\n    </tr>\n    <tr>\n      <th>185</th>\n      <td>6</td>\n    </tr>\n    <tr>\n      <th>286</th>\n      <td>9</td>\n    </tr>\n    <tr>\n      <th>436</th>\n      <td>10</td>\n    </tr>\n    <tr>\n      <th>306</th>\n      <td>10</td>\n    </tr>\n    <tr>\n      <th>17</th>\n      <td>15</td>\n    </tr>\n    <tr>\n      <th>56</th>\n      <td>8</td>\n    </tr>\n    <tr>\n      <th>251</th>\n      <td>10</td>\n    </tr>\n    <tr>\n      <th>253</th>\n      <td>14</td>\n    </tr>\n    <tr>\n      <th>351</th>\n      <td>3</td>\n    </tr>\n  </tbody>\n</table>\n</div>"
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "display(hardness)"
+   ],
+   "metadata": {
+    "collapsed": false
+   }
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "Nie wiem co się stało przy rozpakowywaniu, ale tu są tylko *values*\n",
+    "\n",
+    "*Wydaje mi się, że to też można użyć*\n",
+    "\n",
+    "Wartość specyficzna - głównie dla obiektów -> możliwe, iż powyższe zdanie jest nieaktualne"
```

Comment:
Tu idea jest taka że hardness działa tak jak resistance na wszystkie obrażenia. Czyli u nas można przepisać wartość z hardness do all-damage resistance. 
Ale jest ich też strasznie mało więc można usunąć.

Dodałabym do kategorii na potem i jeśli kiedyś będziemy uwzględniać resistance to możemy przepisać tam wartośći.